onnxruntime/core/providers/openvino/backends/basic_backend.h

```diff
@@ -37,6 +37,7 @@ class BasicBackend : public IBackend {
   SubGraphContext subgraph_context_;
   mutable std::mutex compute_lock_;
   std::shared_ptr<InferenceEngine::CNNNetwork> ie_cnn_network_;
+  InferenceEngine::CNNNetwork cnn_network_;
```

Comment:
We are unnecessarily creating another private variable, which is causing change in both backends. Instead can we just use CreateCNNNetwork function and based on OpenVINO version create the cnnnetwork and give that shared object back ?

Reply:
Done